strr-api/src/strr_api/models/dataclass.py

```diff
@@ -32,7 +32,7 @@
 """ Data Class
 """
 from dataclasses import dataclass
-from typing import List
+from typing import List, Optional
```

Comment:
Optional isn't used anymore, pref is to use |

Reply:
Done